pages/common/ansible-galaxy.md

```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the role id for a role name `{{username.role_name}}`:
```

Comment:
I don't think token syntax in description lines is supported by most clients. How about just
```suggestion
- Get the role id for a role name:
```